modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/parameter/PropagateAllDistributedTraceContextManager.java

```diff
@@ -27,8 +28,7 @@ public PropagateAllDistributedTraceContextManager(DistributedTraceContext distri
   }
 
   private void resolveContextMap(DistributedTraceContext distributedTraceContext) {
-    contextMap = new HashMap<>(distributedTraceContext.tracingFieldsAsMap());
-    contextMap.putAll(distributedTraceContext.baggageItemsAsMap());
+    contextMap = unmodifiableMap(distributedTraceContext.baggageItemsAsMap());
```

Comment:
is this change correct? It seems like previously you were also including the tracing fields but not anymore. Which version is boggus?